src/Export/Adapters/AttributeAdapter.php

```diff
@@ -149,68 +150,24 @@ protected function getManufacturerAttributes(ProductEntity $product): array
     }
 
     /**
+     * @deprecated tag:6.0.0 - Logic was moved to the OptionsAdapter
      * @return Attribute[]
      */
-    protected function getPropertyAttributes(ProductEntity $product): array
+    public function getOptionAttributes(ProductEntity $product): array
     {
-        if (!$product->properties || !$product->properties->count()) {
-            return [];
-        }
-
-        return $this->getPropertyGroupOptionAttributes($product->properties);
+        return [];
     }
 
     /**
      * @return Attribute[]
      */
-    protected function getOptionAttributes(ProductEntity $product): array
+    protected function getPropertyAttributes(ProductEntity $product): array
     {
-        if (!$product->options || !$product->options->count()) {
+        if (!$product->properties || !$product->properties->count()) {
             return [];
         }
 
-        return $this->getPropertyGroupOptionAttributes($product->options);
-    }
-
-    /**
-     * @return Attribute[]
-     */
-    protected function getPropertyGroupOptionAttributes(PropertyGroupOptionCollection $collection): array
-    {
-        $attributes = [];
-
-        foreach ($collection as $propertyGroupOptionEntity) {
-            $group = $propertyGroupOptionEntity->group;
-            if ($group && !$group->filterable) {
-                continue;
-            }
-
-            $attributes = array_merge($attributes, $this->getAttributePropertyAsAttribute($propertyGroupOptionEntity));
-        }
-
-        return $attributes;
-    }
-
-    /**
-     * @return Attribute[]
-     */
-    protected function getAttributePropertyAsAttribute(PropertyGroupOptionEntity $propertyGroupOptionEntity): array
-    {
-        $attributes = [];
-
-        $group = $propertyGroupOptionEntity->group;
-        if ($group && $propertyGroupOptionEntity->getTranslation('name') && $group->getTranslation('name')) {
-            $groupName = $this->getAttributeKey($group->getTranslation('name'));
-            $propertyGroupOptionName = $propertyGroupOptionEntity->getTranslation('name');
-            if (!Utils::isEmpty($groupName) && !Utils::isEmpty($propertyGroupOptionName)) {
-                $propertyGroupAttrib = new Attribute($groupName);
-                $propertyGroupAttrib->addValue(Utils::removeControlCharacters($propertyGroupOptionName));
-
-                $attributes[] = $propertyGroupAttrib;
-            }
-        }
-
-        return $attributes;
+        return $this->getPropertyGroupOptionAttributes($product->properties, $this->pluginConfig);
```

Comment:
This is how it should look like, when having a trait with the member variable.

```suggestion
        return $this->getPropertyGroupOptionAttributes($product->properties);
```